README.md

```diff
@@ -1,50 +1,88 @@
 # OpenCBM
-Win 7/8/10, and Linux/i386/AMD64 kernel driver and development library to control serial CBM devices, such as the Commodore 1541 disk drive, connected to the PC's parallel port via a XM1541 or XA1541 cable. Fast disk copier included. Successor of cbm4linux. Also supports the XU1541 and the XUM1541 devices (a.k.a. "ZoomFloppy").
+
+Win 7/8/10, and Linux/i386/AMD64 kernel driver and development library to
+control serial CBM devices, such as the Commodore 1541 disk drive, connected to
+the PC's parallel port via a XM1541 or XA1541 cable. Fast disk copier included.
+Successor of cbm4linux. Also supports the XU1541 and the XUM1541 devices
+(a.k.a. "ZoomFloppy").
 
```

Comment:
Why do you believe a line of 80 characters is better readable? Isn't it up to the user to decide how long he wants his output to be?